config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -168,13 +177,20 @@ periodics:
         repo:kubernetes/test-infra
       - --token=/etc/github-token/token
       - |-
-        --comment=/retest
-        This bot automatically retries jobs that failed/flaked on approved PRs (send feedback to [fejta](https://github.com/fejta)).
+        --comment=The Kubernetes project has merge-blocking tests that are too flaky to consistently pass.
```

Comment:
if we're going to make this kind of statement, then we need to add something about what the submitter could do about such flaky tests.  e.g. "You can help improve test consistency by contributing to <location>"

It's also overly negative; even if our tests were 100% the best they could possibly be, the nature of cloud testing is such that we'd get a small number of essentially random failures.

Reply:
>if we're going to make this kind of statement, then we need to add something about what the submitter could do about such flaky tests. e.g. "You can help improve test consistency by contributing to {location}"

I will see if I can add a link below to our [Flaky Tests Guide][1]

>It's also overly negative; 

It's the state of today.  Some back of the napkin math I did [over here][1] was that at least as of 8 days ago, you had a 66% chance of a kubernetes/kubernetes typo-fix PR passing tests on the first try.  Better than a coin flip!  But probably not a developer's expectations of the phrase "consistently pass"

>even if our tests were 100% the best they could possibly be, the nature of cloud testing is such that we'd get a small number of essentially random failures.

Perhaps we should evict cloud testing from our workflow!  But that's a whole other tangent.  I agree 100% is unrealistic, and not what I meant to imply with "consistently pass".  What would you suggest instead?

[1]: https://github.com/kubernetes/test-infra/issues/6380#issuecomment-887173912
[2]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/flaky-tests.md

Reply:
link added in https://github.com/kubernetes/test-infra/pull/23094/commits/b3868685873619553820569f8e7d3be147dfcce3

Reply:
We actually do get near 100% consistency in `pull-kubernetes-e2e-kind` for the majority of this release (and in e.g. unit testing, build, linters ...), evicting cloud from presubmit is on the table IMHO but not something we should decide here or reasonable for PR authors to be worrying about. I think maybe linking to the deflaking guide or pointing them to e.g. sig-testing would be sufficient.

We probably should actually be talking about improving this ourselves including by possibly evicting cloudy presubmits.